src/routes.py

```diff
@@ -28,11 +37,34 @@ def registration() -> str:
         user.set_password(form.password.data)
         session.add(user)
         session.commit()
+        return redirect(url_for("routes.login"))
```

Comment:
Yep, I know why added this, but it's not in the scope of current task, so It's better to do that in another ticket/hotfix.

If you really need this redirect functionality, I can add this as a hotfix to `develop`.